smack-experimental/src/main/java/org/jivesoftware/smackx/chat_markers/ChatMarkersManager.java

```diff
@@ -144,8 +145,13 @@ public void processStanza(Stanza packet)
                 // Note that this listener is used together with a PossibleFromTypeFilter.ENTITY_BARE_JID filter, hence
                 // every message is guaranteed to have a from address which is representable as bare JID.
                 EntityBareJid bareFrom = message.getFrom().asEntityBareJidOrThrow();
+                Chat tempChat = null;
+                try {
+                    tempChat = chatManager.chatWith(bareFrom);
+                } catch (Exception e) {
```

Comment:
Please do not swallow exceptions silently.

Reply:
`catch (Exception e)` is also bad practice and consider code smell unless it is one of the rare valid cases where you really want to catch all `Exception`s.

Reply:
True is: I didn't went deeper in `chatWith()` method to search for all exceptions that can be thrown 😅, but the idea around it is because current code doesn't allow to be used in MUC (MUC use old `Chat` v1) so if I try to get a Chat (v2) giving a Bare from a room (group chat) it's not gonna work... in fact MUC doesn't needs the Chat.

So I can dig into that method to look for specific exceptions but I think (I'm not sure, you are the guys that know it better than me) the`empty try catch` is needed.

Tell me what to do and I will do it.

Reply:
Simply do not catch any exception, there is no point in what the code actually does. It is actually harmful, because the `run()` method below will possibly be invoked with `chat == null`.